neo/public/javascripts/common/services.js

```diff
@@ -1,12 +1,37 @@
 angular.module('cloudberry.common', [])
   .service('Asterix', function($http, $timeout, $location) {
     var startDate = new Date(2015, 10, 22, 0, 0, 0, 0);
+    var defaultNonSamplingDayRange = 1500;
+    var defaultSamplingDayRange = 1;
+    var defaultSamplingSize = 10;
     var ws = new WebSocket("ws://" + $location.host() + ":" + $location.port() + "/ws");
 
-    ws.onopen = function() {
-      var json = JSON.stringify({ cmd : "totalCount"});
-      ws.send(json);
-    };
+    var countRequest = JSON.stringify({
```

Comment:
I see, here is the place that causes the confusion.
The request should be like this
```
    transform: {
         wrap: {
                  key : "totalCount",
                  value : "value"  // means the returned result is in the `value` field
         }
     }
     // all the original fields
 
     dataset: "twitter.ds_tweet",
      global: {
            globalAggregate: {
               field: "*",
               apply: {
                 name: "count"
                },
                as: "count"
           }},
           estimable : true
          }
        }

```